fs/accounting/stats.go

```diff
@@ -13,8 +14,13 @@ import (
 	"github.com/rclone/rclone/fs/rc"
 )
 
-// MaxCompletedTransfers specifies maximum number of completed transfers in startedTransfers list
-var MaxCompletedTransfers = 100
+var (
+	// MaxCompletedTransfers specifies maximum number of completed transfers in startedTransfers list
+	MaxCompletedTransfers = 100
+
+	// SpeedRingSize specifies the size of duration and speed rings for computing the rolling average
+	SpeedRingSize = 16
```

Comment:
The other speed transfer measures in rclone use [exponentially weighted averages](https://en.wikipedia.org/wiki/Moving_average#Exponential_moving_average) which work very well and are a lot easier to calculate as you don't need to store the history.

https://github.com/rclone/rclone/blob/68afa28b27f922f1cdf5e872549a870ff2a84401/fs/accounting/accounting.go#L179-L184

Reply:
Yes, using that seems to be a good idea. But according to my experience, it decays too slowly when there is a sudden drop. What I want to achieve is, when the network or the remote service gets stuck, the speed drops to zero in a few seconds.

Reply:
Hi, do you think something like this still worth pursuing? Or should I just (re)use the existing exponentially weighted average here?

Reply:
Let's try the exponentially weighted average as it is a lot simpler - we can then consider whether we want to switch it out.

Sorry for the delay in responding!